### PR TITLE
Correct varying varchar

### PR DESCRIPTION
**Pull request by sebCIL**
### Changes

Correct varying varchar

### Checklist

* [X] have tested my change
* [ ] updated relevant documentation
* [X] Remove any/all `console.log`s I added
* [ ] eslint is not complaining
* [ ] have added myself to the contributors' list in the README
* [ ] **for feature PRs**: PR only includes one feature enhancement.

### CODE REVIEW

File: server/src/server.ts [hunk 1]
```diff
@@ -121,9 +121,9 @@ parser.setTableFetch(async (table: string, aliases = false) => {
 
 		currentSubfield.keywords = [getPrettyType({
 			type,
-			len: digits === 0 ? strLength : digits,
+			len: digits === `0` ? strLength : digits,
```
Comment (worksofliam) [inline server/src/server.ts:124]: I am not sure this is right.

`digits` is always a number and a strict comparison (`===`) against `'0'` will always return false.

Comment (worksofliam) [reply server/src/server.ts:124]: image

Comment (worksofliam) [reply server/src/server.ts:124]: Another example digits is a number. I can confirm `digits` is always a number and so this change could be wrong. 
image

**Comment by sebCIL** [comment]
But digits was a String and the value is casting into a number.

In this exemple, we can see that the length in getPrettyType is 0 while the length was 52
2022-10-28_17h24_14
2022-10-28_17h24_46

With the modification
2022-10-28_17h40_33
2022-10-28_17h41_45

**Comment by worksofliam** [comment]
@sebCIL Ok, I have found a bug in `vscode-ibmi`. I need to make a fix and release before we can merge this.

With SQL disabled, these are strings. With SQL enabled, these are numbers. They should always be numbers.

```
			WHFLDB: strLength, 
			WHFLDD: digits,
```

**Comment by worksofliam** [comment]
@sebCIL vscode-ibmi@1.5.22 has just been released. Update the extension, debug again and you should always see numbers this time.

It returning a string was a bug, and I have now fixed that.

**Comment by sebCIL** [comment]
Waouh ! That's the definition of reactivity ! Thanks. :clap:
It look good. I remove the it.

**Comment by worksofliam** [comment]
@sebCIL Awesome fixes. Thank you!

**Pull request merged**